### DISCUSSION TITLE
api.wyzecam.com getting pinged every second

### DISCUSSION THREAD

**Discussion by huntrm** [SecKatie/ha-wyzeapi#436] [Q&A]
My pihole is reporting that api.wyzecam.com is getting pinged every second.  Is this normal behavior for the integration?  Thank you.

**Comment by treykane** [comment]
I've also noticed this behavior and was unsure if it was normal.

**Comment by JoeSchubert** [comment]
It's intended for it to be pinged about once per second per device. Currently, it's pinging more than intended as can be seen in this issue thread https://github.com/JoshuaMulliken/ha-wyzeapi/issues/441

Trust me when I say that the app floods their API far more than this integration ever could (in it's current state). You just probably don't see it in your dns resolver due to ssl pinning, and likely dns pinning, in the app.
